.coderabbit.yml

```
@@ -0,0 +1,2 @@
+reviews:
+  path_filters: ["**/*","*.*"]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)